ros_ws/src/simulation/racer_control/include/drive_param_converter.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <drive_msgs/drive_param.h>
+#include <std_msgs/Float64.h>
+
+/**
+ * @brief 
+ * 
+ */
+class DriveParamConverter
+{
+    public:
+    /**
+     * @brief Construct a new Drive Pram Converter object
+     * 
+     */
+    DriveParamConverter();
+    /**
+     * @brief 
+     * 
+     * @param parameters 
+     */
+    void convertDriveParametersCallback(const drive_msgs::drive_param::ConstPtr& parameters);
+    
+
+    private:
+    /**
+     * @brief 
+     * 
+     */
+    struct Angles
+    {
+        double angle1;
+        double angle2;
+    };
+    /**
+     * @brief 
+     * 
+     * @param angle Angle of the center of the front axis.
+     * @return Angles One Ackermann angle for each front wheel
+     */
+    Angles calculateSteeringAngles(const double& angle);
+    /**
+     * @brief 
+     * 
+     */
+    ros::NodeHandle m_node_handle;
+    /**
+     * @brief 
+     * 
+     */
+    ros::Subscriber m_drive_parameters_subscriber;
+    /**
+     * @brief 
+     * 
+     */
+    ros::Publisher m_left_rear_wheel_velocity_publischer;
+    /**
+     * @brief 
+     * 
+     */
+    ros::Publisher m_right_rear_wheel_velocity_publischer;
+    /**
+     * @brief 
+     * 
+     */
+    ros::Publisher m_left_front_wheel_velocity_publischer;
+    /**
+     * @brief 
+     * 
+     */
+    ros::Publisher m_right_front_wheel_velocity_publischer;
+    /**
+     * @brief 
+     * 
+     */
+    ros::Publisher m_left_steering_position_publischer;
+    /**
+     * @brief 
+     * 
+     */
+    ros::Publisher m_right_steering_position_publischer;
+    
+    /**
+     * @brief 
+     * 
+     */
+    const double m_length;
+
+    /**
+     * @brief 
+     * 
+     */
+    const double m_width;
```

Comment:
I assume that these refer to the length and width of the car, not the `DriveParamConverter`. They should be named accordingly, for example `CAR_WIDTH`. As suggested by @Dadie, this should be initialized here.